frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToRemote.java

```diff
@@ -184,21 +198,39 @@ private boolean handleAheadOfRemote(IManagedBranchSnapshot gitMacheteBranch, Git
 
   @ContinuesInBackground
   @UIEffect
-  private boolean handleDivergedFromRemote(IManagedBranchSnapshot gitMacheteBranch,
-      SyncToRemoteStatus syncToRemoteStatus,
-      GitLocalBranch localBranch) {
+  private boolean handleDivergedFromAndNewerThanRemote(IManagedBranchSnapshot gitMacheteBranch, GitLocalBranch localBranch) {
     val remoteTrackingBranch = gitMacheteBranch.getRemoteTrackingBranch();
     assert remoteTrackingBranch != null : "remoteTrackingBranch is null";
     val selectedAction = new DivergedFromRemoteDialog(project, remoteTrackingBranch, gitMacheteBranch,
```

Comment:
hmmmm 🤔 I thought we can drop the `DivergedFromRemoteDialog` class and use yesnocancel instead. right?

Reply:
#1428